src/mame/includes/jedi.h

```diff
@@ -46,7 +46,7 @@ class jedi_state : public driver_device
 		m_sacklatch(*this, "sacklatch"),
 		m_tms(*this, "tms"),
 		m_novram(*this, "novram12%c", 'b'),
-#ifdef DEBUG_GFXDECODE
+#if DEBUG_GFXDECODE
```

Comment:
Is this guaranteed to be defined?  You’ll get a warning for testing an undefined macro if it isn’t.

Reply:
Yes. The header currently has #define DEBUG_GFXDECODE (0) which is clearly meant to turn off the DEBUG_GFXCODE but instead was leaving it on because everything was using #ifdef.